soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -963,18 +956,18 @@ static const char* windowType[] = { "Floating", "Window" };
 static const char* displayType[] = { "Always", "Combo Button Hold" };
 static const char* buttonStrings[] = { "A Button", "B Button", "C-Up",  "C-Down", "C-Left", "C-Right", "L Button",
                                        "Z Button", "R Button", "Start", "D-Up",   "D-Down", "D-Left",  "D-Right" };
-void DrawCheckTrackerOptions(bool& open) {
-    if (!open) {
```

Comment:
This should be removed (unless this window should be different than others for some reason)

Reply:
is this https://github.com/HarbourMasters/Shipwright/pull/2881/commits/ef399ee8c0d72805e3a31222456dbfdd21eccf7e what you wanted removed? (hard to tell because your comment is right under 2 deleted lines)

Reply:
Yep that was it